compiler/src/iree/compiler/Utils/ElementPackingUtils.cpp

```diff
@@ -113,12 +140,15 @@ Value calculateStorageElementOffsetInBytes(Location loc,
                                            RankedTensorType originalType,
                                            Value linearizedIndex,
                                            OpBuilder &builder) {
-  Type alignedElementType =
-      legalizeStorageElementType(originalType.getElementType());
+  // TODO: remove cl options once frontend can emit packed i1 tensors.
```

Comment:
We do not want to duplicate the comment. I think we can move the TODO to where the cl option is defined, and remove this one.